FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+
+
+  /// Checks if a custom user interface is desired by the end developer. Returns TRUE if using a custom UI, FALSE if using the default.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT SuppressDefaultFindDialog([in] BOOL value);
+
+
+
+  /// Adds an event handler for the `ActiveMatchIndexChanged` event.
+  /// Registers an event handler for the ActiveMatchIndexChanged event. This event is raised when the index of the currently active match changes. This can happen when the user navigates to a different match or when the active match is changed programmatically. The parameter is the event handler to be added. Returns a token representing the added event handler. This token can be used to unregister the event handler.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT add_ActiveMatchIndexChanged(
+      [in] ICoreWebView2StagingActiveMatchIndexChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_ActiveMatchIndexChanged`.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT remove_ActiveMatchIndexChanged(
+      [in] EventRegistrationToken token);
+
+  /// Adds an event handler for the `MatchCountChanged` event.
+  /// Registers an event handler for the MatchCountChanged event. This event is raised when the total count of matches in the document changes due to a new find operation or changes in the document.    /// The parameter is the event handler to be added. Returns a token representing the added event handler. This token can be used to unregister the event handler.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT add_MatchCountChanged(
+      [in] ICoreWebView2StagingMatchCountChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_MatchCountChanged`.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT remove_MatchCountChanged(
+      [in] EventRegistrationToken token);
+
+
+  /// Initiates a find using the specified configuration.
+  /// Displays the Find bar and starts the find operation. If a find session was already ongoing, it will be stopped and replaced with this new instance.
+  /// If called with an empty string, the Find bar is displayed but no finding occurs. Changing the configuration object after initiation won't affect the ongoing find session.
+  /// To change the ongoing find session, StartFind must be called again with a new or modified configuration object.
+  /// This method is primarily designed for HTML document queries.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT StartFind(
+      [in] ICoreWebView2StagingFindConfiguration* configuration
+      , [in] ICoreWebView2StagingFindStartFindCompletedHandler* handler
+  );
+
+  /// Navigates to the next match in the document.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT FindNext(
```

Comment:
If the FindConfiguration FindDirection is set to Backwards, FindNext will find the match that occurs before the current match - is that correct?
We should explicitly document the relation between FindNext/FindPrevious and FindDirection.